blog/tidy-finance-dummy-data/index.qmd

```diff
@@ -0,0 +1,422 @@
+---
```

Comment:
General remark: Should we add this to our book's appendix? Maybe only the code itself?

Reply:
Out of scope for this PR, we need to discuss this later.